src/App.js

```diff
@@ -0,0 +1,235 @@
+  // "C605.wav",
+  // "Cs507.wav",
+  // "D502.wav",
+  // "D605.wav",
+  // "  Ds601.wav",
+  // " E503.wav",
+  // "  E604.wav",
+  // " F507.wav",
+  // " F608.wav",
+  // "  G501.wav",
+  // "  Gs502.wav",
+  // "A506.wav",
+  // "     As503.wav",
+  // "    B406.wav",
+  // " C502.wav",
+  // " C606.wav",
+  // "  Cs601.wav",
+  // " D503.wav",
+  // "  D606.wav",
+  // "  Ds602.wav",
+  // " E504.wav",
+  // "  E605.wav",
+  // " F508.wav",
+  // " F609.wav",
+  // "  G502.wav",
+  // "  Gs503.wav",
+  // "A507.wav",
+  // "     As504.wav",
+  // "    B501.wav",
+  // " C503.wav",
+  // " C607.wav",
+  // "  Cs602.wav",
+  // " D504.wav",
+  // "  D607.wav",
+  // "  Ds603.wav",
+  // " E505.wav",
+  // "  E606.wav",
+  // " F509.wav",
+  // " Fs501.wav",
+  // " G503.wav",
+  // "  Gs504.wav",
+  // "A508.wav",
+  // "     As505.wav",
+  // "    B502.wav",
+  // " C504.wav",
+  // " C608.wav",
+  // "  Cs603.wav",
+  // " D505.wav",
+  // "  D608.wav",
+  // "  Ds604.wav",
+  // " E506.wav",
+  // "  E607.wav",
+  // " F601.wav",
+  // " Fs502.wav",
+  // " G504.wav",
+  // "  Gs505.wav",
+  // "As4-B401.wav",
+  // " As506.wav",
+  // "    B503.wav",
+  // " C505.wav",
+  // " Cs501.wav",
+  // " Cs604.wav",
+  // " D506.wav",
+  // "  Ds501.wav",
+  // " Ds605.wav",
+  // " E507.wav",
+  // "  F501.wav",
+  // " F602.wav",
+  // " Fs503.wav",
+  // " G505.wav",
+  // "  Gs506.wav",
+  // "As4-B402.wav",
+  // " As507.wav",
+  // "    B504.wav",
+  // " C506.wav",
+  // " Cs502.wav",
+  // " Cs605.wav",
+  // " D507.wav",
+  // "  Ds502.wav",
+  // " Ds606.wav",
+  // " E508.wav",
+  // "  F502.wav",
+  // " F603.wav",
+  // " Fs504.wav",
+  // " G506.wav",
+  // "  Gs507.wav",
+]
+
+const pianoAudioFiles = [
+  '203459__tesabob2001__a-5.mp3',
+  '203481__tesabob2001__c-4.mp3',                 
+  '203460__tesabob2001__a-4.mp3',         
+  '203482__tesabob2001__d-4.mp3',          
+  '203482__tesabob2001__d-4.mp3',          
+  '203483__tesabob2001__d-3.mp3',                    
+  '203462__tesabob2001__b4.mp3',          
+  '203484__tesabob2001__c6.mp3' ,                   
+  '203463__tesabob2001__b3.mp3',          
+  '203485__tesabob2001__c5.mp3' ,                   
+  '203464__tesabob2001__a5.mp3',          
+  '203486__tesabob2001__d3.mp3' ,                   
+  '203465__tesabob2001__a4.mp3',          
+  '203487__tesabob2001__d-5.mp3',                                 
+  '203469__tesabob2001__f4.mp3',          
+  '203491__tesabob2001__g-4.mp3',                    
+  '203470__tesabob2001__e3.mp3',          
+  '203492__tesabob2001__g4.mp3' ,                   
+  '203471__tesabob2001__e4.mp3',          
+  '203493__tesabob2001__g3.mp3' ,                   
+  '203472__tesabob2001__d4.mp3',           
+  '203472__tesabob2001__d4.mp3',           
+  '203473__tesabob2001__d5.mp3',          
+  '203495__tesabob2001__g5.mp3 ',                    
+  '203474__tesabob2001__f-4.mp3',         
+   '203499__tesabob2001__f-5.mp3'  
+]
```

Comment:
I'm always a fan of moving data from main files to separate files and importing them. It helps for readability.